react/src/base/typography/SprkHeading/SprkHeading.stories.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import SprkHeading from './SprkHeading';
+import SprkText from '../SprkText/SprkText';
+
+import { markdownDocumentationLinkBuilder } from '../../../../../storybook-utilities/markdownDocumentationLinkBuilder';
+
+export default {
+  title: 'Foundations/Typography',
+  decorators: [
+    story => <div className="sprk-o-Box">{story()}</div>
+  ],
+  component: SprkHeading,
+  subcomponents: {
+    SprkText,
+  },
+  parameters: {
+    jest: ['SprkHeading'],
+    info: `
+${markdownDocumentationLinkBuilder('typography', 'foundations')}
+
+##### Line-height Collision
+
+By default, line height on type will create extra
```

Comment:
line-height

Reply:
@RCopeland , we have this on the gatsby site like I have it in this PR. https://sparkdesignsystem.com/using-spark/foundations/typography/#gatsby-focus-wrapper - I'm going to keep it like we have it in the gatsby site.